lib-ee/emqx_ee_bridge/src/emqx_ee_bridge_gcp_pubsub.erl

```diff
@@ -79,15 +79,6 @@ fields(bridge_config) ->
                     desc => ?DESC("max_retries")
                 }
             )},
-        {request_timeout,
```

Comment:
should deprecate? removing it will cause compatibility issue. (if it's released in e5.0.0)

Reply:
It's released there, indeed.  I'll change it to deprecation.

Reply:
:heavy_check_mark: